.storybook/blocks/PropertiesTable.jsx

```diff
@@ -0,0 +1,66 @@
+import "@spectrum-css/table";
+
+import { DocsContext, useOf } from "@storybook/blocks";
+import { ResetWrapper } from "@storybook/components";
+import { NAVIGATE_URL } from "@storybook/core-events";
+import { styled } from "@storybook/theming";
+import React, { useContext } from 'react';
+import { Body, Code, LinkableHeading } from "./Typography.jsx";
+
+export const Table = styled.table`
+	padding-block: 10px;
+`;
+
+/**
+ * Displays the current version number of the component. The version is read from
+ * the component's parameters, where it was sourced from the package.json file.
+ *
+ * Also displays a component status of "deprecated" if it is set in the story's
+ * parameters.
```

Comment:
I think this might be the `ComponentDetails` description?

Reply:
Yes! Good catch. Fixed this in my latest update.